pkg/rbac/parser.go

```diff
@@ -48,6 +48,10 @@ type Rule struct {
 	Groups []string `marker:",optional"`
 	// Resources specifies the API resources that this rule encompasses.
 	Resources []string `marker:",optional"`
+	// ResourceNames specifies the names of the API resources that this rule encompasses.
+	// Create requests cannot be restricted by resourcename, as the object's name
+	// is not known at authorization time.
```

Comment:
Nit:

```suggestion
	// ResourceNames specifies the names of the API resources that this rule encompasses.
	//
	// Create requests cannot be restricted by resourcename, as the object's name
	// is not known at authorization time.
```

This'll put the extra `Create requests...` in the details field of the docs.